src/engine/guidance/post_processing.cpp

```diff
@@ -821,6 +827,7 @@ std::vector<RouteStep> collapseTurns(std::vector<RouteStep> steps)
         else if (isCollapsableInstruction(current_step.maneuver.instruction) &&
                  current_step.maneuver.instruction.type != TurnType::Suppressed &&
                  steps[getPreviousNameIndex(step_index)].name_id == current_step.name_id &&
+                 // canCollapseAll is also checking for compatible turns
```

Comment:
What do you mean by this? compatible checks the travel mode, but what do you mean with compatible turns? from one travel mode to another I suppose?